moveit_core/collision_detection_fcl/src/collision_common.cpp

```diff
@@ -50,6 +50,8 @@
 #include <boost/thread/mutex.hpp>
 #include <memory>
 
+rclcpp::Logger logger = rclcpp::get_logger("collision_detection.fcl");
```

Comment:
```suggestion
const rclcpp::Logger LOGGER = rclcpp::get_logger("collision_detection.fcl");
```

Reply:
The issue has been addressed on the following https://github.com/ros-planning/moveit2/pull/41#discussion_r286228511